langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -100,10 +108,11 @@ public AzureOpenAiChatModel(OpenAIClient client,
                                 List<AzureChatExtensionConfiguration> dataSources,
                                 AzureChatEnhancementConfiguration enhancements,
                                 Long seed,
-                                ChatCompletionsResponseFormat responseFormat,
```

Comment:
This is a breaking change, some people have `.responseFormat(new ChatCompletionsJsonResponseFormat())` in their code. Can we keep supporting `ChatCompletionsResponseFormat`? We can make it deprecated in the builder though.

Reply:
As we also add the `boolean strictJsonSchema` argument just after, there will be a breaking change in the constructor anyway: are you fine if we only do this in the Builder? It would accept `ChatCompletionsResponseFormat` (with a deprecation notice) and the `ResponseFormat`.

Reply:
I reverted to the former API